proposed/attributes-meta.md

```diff
@@ -0,0 +1,70 @@
+# Attributes Meta Document
+
+## 1. Summary
+
+This Working Group aims to create and manage a shared set of PHP attributes that may be used by many different actors within the PHP ecosystem, avoiding duplication and redundancy.
+
+## 2. Why Bother?
+
+Right now libraries that want to use attributes either have to define them themselves or rely upon 
+another library that has them defined. Most of the time that also means to import all the code of 
+the other library. 
+
+Userland libraries have no interest in provising an interoperable set of attributes as they solve 
+a specific problem with *their* set of attributes. 
+
+This registry shall solve that
```

Comment:
This feels like an unfinished sentence? Maybe rephrase to:

An intropable shared registry with attribute definitions will solve this.